.github/workflows/prepare-release.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           body-path: changes-since-last-tag.txt
           labels: no-jira-ticket
           commit-message: Prepare for release ${{ inputs.version }}
-          token: ${{ secrets.REALM_CI_PAT }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't think we need any of the special abilities of the REALM_CI_PAT for this action. We should be fine with the default GITHUB_TOKEN ([docs](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow)). One difference is that further github actions cannot be triggered by jobs using the GITHUB_TOKEN, but this is not an issue because the next step in the release process is to approve and manually merge the generated PR.

Reply:
That's right - if you only work in the same repository and don't need subsequent workflows to trigger for any of the work `GITHUB_TOKEN` does then I'd say it's the best choice.

Reply:
Thanks for confirming Yavor!